packages/block-library/src/cover/index.js

```diff
@@ -429,6 +412,9 @@ export const settings = {
 	deprecated: [ {
 		attributes: {
 			...blockAttributes,
+			align: {
```

Comment:
This is not necessary. Behind the scenes, the `supports` flag adds the `align` attribute to the block, so the block still has the same attribute set regardless of whether `align` is manually defined or the `supports` flag is used.